test/vcf/debugulator_integration_test.cpp

```diff
@@ -52,79 +52,124 @@ namespace ebi
       return validator.is_valid();
   }
 
-  TEST_CASE("Integration test: Fixing a VCF with duplicates", "[debugulator]")
+  std::tuple<bool, bool, std::string, long> fix_file(boost::filesystem::path path, vcf::Version version)
   {
-      auto path = boost::filesystem::path("test/input_files/v4.1/failed/failed_body_duplicated_000.vcf");
-      SECTION(path.string())
-      {
-          std::ifstream file{path.c_str()};
-          if (not file) {
-              throw std::runtime_error{"test file not found: " + path.string()};
+      std::ifstream file{path.c_str()};
+      if (not file) {
+          throw std::runtime_error{"test file not found: " + path.string()};
```

Comment:
The test should fail anyway if the file doesn't exist. If it doesn't, then it's an issue that this exception would hide.